tests/installation/grub_test.pm

```diff
@@ -73,10 +73,12 @@ sub run {
             send_key "ret";
         }
     }
-    workaround_type_encrypted_passphrase;
-    # 60 due to rare slowness e.g. multipath poo#11908
-    assert_screen "grub2", 60;
-    stop_grub_timeout;
+    else {
+        workaround_type_encrypted_passphrase;
```

Comment:
Now this only gets executed if STORAGE_NG and ENCRYPT are "false". Can't you just move the workaround into l#65 ?

Reply:
These three lines would be executed if STORAGE_NG **OR** ENCRYPT are "false", which I think is like to say, "the rest of the options". To be more explicit it could be checked `elsif(!get_var("STORAGE_NG") && !get_var("ENCRYPT"))`. But I don't see why to move to l#65 `workaround_type_encrypted_passphrase`, because I would break the order and then it would not run for the rest. Initially was set to do it always. I believe that we need to exclude this part for this test and let the rest as it was for backcompatibility.

Reply:
I think you mean l#75 to not break the order, after the foreach, right?